package.json

```diff
@@ -296,6 +297,7 @@
     "bignumber.js": "^4.1.0",
     "bn.js": "^4.11.7",
     "bowser": "^2.11.0",
+    "browserify-zlib": "^0.2.0",
```

Comment:
Where is this new dependency used `browserify-zlib`

Reply:
A bunch of changes have been reverted already, I had previously merged another PR by mistake that brought unneeded changes. 🤦🏼